lib/components/intent.js

```diff
@@ -535,6 +535,49 @@ Intent.prototype.setPresence = function(presence, status_msg=undefined) {
     });
 };
 
+// type BridgeErrorReason =
+//    |"m.event_not_handled"
+//    |"m.event_too_old"
+//    |"m.internal_error"
+//    |"m.foreign_network_error"
+//    |"m.event_unknown"
+
+/**
+ * Signals that an error occured while handling an event by the bridge.
+ * @param {string} roomId ID of the room in which the error occured.
+ * @param {string} eventId ID of the event for which the error occured.
+ * @param {string} network_name Name of the bridged network.
+ * @param {BridgeErrorReason} reason The reason why the bridge error occured.
+ * @param {string[]} affected_users Optional. Matrix IDs of all affected users.
+ * @return {Promise}
+ */
+Intent.prototype.signalBridgeError = function(
+    roomId,
+    eventId,
+    network_name,
+    reason,
+    affected_users
+) {
+    if (!affected_users) {
```

Comment:
Interesting. Could we save some bytes here and have affected users be `["*"]`. It feels a bit nicer than trying to fetch all the current users.